code/game/objects/random/random.dm

```diff
@@ -514,7 +514,7 @@
 /obj/random/tank/spawn_choices()
 	return list(/obj/item/tank/oxygen = 5,
 				/obj/item/tank/oxygen/yellow = 4,
-				/obj/item/tank/oxygen/red = 4,
+				/obj/item/tank/emergency/oxygen/double/red = 4,
```

Comment:
You could honestly include both sizes of red tanks in random spawns

Reply:
Na the normal red tank shares the same sprite as the nitrogen tanks. Don't wanna mix them up